index.html

```diff
@@ -54,164 +62,153 @@ <h2>LET'S CONNECT</h2>
-        <div class="card-info">
-        <div class="card-image">
-          <img src="images/Portfolio.png" alt="">
-        </div>
-        <div class="card-header">
-          <h2>Tonic</h2>
-        </div>
-        <ul class="card-details">
-          <li class="card-first-element">CANOPY</li>
-          <li class="circle-icon"><i class="fa fa-circle"></i></li>
-          <li class="card-second-element">Back End Dev</li>
-          <li class="circle-icon"><i class="fa fa-circle"></i></li>
-          <li class="card-third-element">2015</li> 
-          </ul>
-        <div class="card-content">
-          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-        </div>
-        <ul class="btn-button">
-          <li class="btn-languages"><a href="#" class="btn-link">html</a></li>
-          <li class="btn-languages"><a href="#" class="btn-link">css</a></li>
-          <li class="btn-languages"><a href="#" class="btn-link">javascript</a></li>
-        </ul>
-        <button class="btn-card">See project</button>
-        </div>
-      </section>
-      <!--end of work section-->
-
-      <!--start about me section-->
-     <section class="about_me">
-        <div class="about_container">
-          <h3 class="about_me_title"> About <br>Myself</h3>
-          <div class="about_info">
-            <h3 class="about_me_title-desktop">About me</h3>
-            <div class="about_description">
-              <p>
-                Hello I’m a software developer! I can help you build a product ,
-                feature or website Look through some of my work and experience! If
-                you like what you see and have a project you need coded, don’t
-                hestiate to contact me.
-              </p>
-            </div>
-            <div>
-              <button type="button" class="about_btn">Get My Resume</button>
+              <!--contact form-->
+           <form action="https://formspree.io/f/xknykvyd" method="post" class="form-container">
+              <div class="container-form">
+                <h2 class="form-header">Contact Me</h2>
+                <p class="form-content">if you have an application you are interested in developing a feature that you need in coding.
+                  I'd love to help with it
+                </p>
+                    <label for="name">
+                    <input type="text" id="name" class="text-input" name="user-name" placeholder="Abdullah" maxlength="30" required>
+                  </label>
+                    <label for="email">
+                    <input type="email" name="user-email" id="email" class="text-input" placeholder="muhammadiyann@gmail.com" required>
+                   </label>
+                <textarea name="message" id="msg" class="text-area" placeholder="Write your Message here" maxlength="500" required></textarea>
+            <button type="submit" class="btn-submit">Get in touch</button>
```

Comment:
- [ ] Please give a padding of `16px` to your input fields so that they are not glued to the left border when typing 👍 
- [ ] Kindly remove the border on your input fields so they look more like the given Figma design
- [ ] I noticed that your input fields are not quite as wide as your `textarea `which makes it look unbalanced. To fix this, simply give them a width to match that of the `textarea `in order to have a balanced design 🙏 
- [ ] Please center your button just like the one in the Figma design

As a reference, please see the images below 👇 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/168427359-86dbee43-841e-47cc-a05a-4a222507862b.png)
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/168427442-a79dce9b-9596-412a-a7e0-b628b1ed4496.png)